trackbasket_be/models/krogerservice.py

```diff
@@ -47,4 +48,47 @@ def closest_store(cls, zipcode):
               'latitude': parsed_response['data'][0]['geolocation']['latitude'],
               'longitude': parsed_response['data'][0]['geolocation']['longitude']            
             }
+  
+  @classmethod
+  def item_search(cls, term, at_risk_user_id):
+    
+    user = AtRiskUser.find_by_id(at_risk_user_id) 
+    nearest_store = user.store[0]
+    nearest_store_id = nearest_store.location_id
+    parameters = {'filter.term': term, 'filter.locationId': nearest_store_id}
+    headers = {'Authorization': 'Bearer {}'.format(Krogerservice.return_token())}
+    response = requests.get('https://api.kroger.com/v1/products', params=parameters, headers=headers)
+
+    json_items = response.json()['data']
+    
+    items = []   
+      
+    for item in json_items:
+      data = {
```

Comment:
add prefilled array for consistency with item model

Reply:
for readability, we could replace that by `data = {}`